### PR TITLE
Silence pytest warning

### PR DESCRIPTION
**Pull request by z3ntu**
```
tests/test_connection.py:34
  /tmp/dasbus/tests/test_connection.py:34: PytestCollectionWarning: cannot collect test class 'TestMessageBus' because it has a __init__ constructor (from: tests/test_connection.py)
    class TestMessageBus(MessageBus):
```

### CODE REVIEW

**Comment by poncovka** [comment]
The errors should be fixed by https://github.com/rhinstaller/dasbus/pull/43 now.

**Review by poncovka — CHANGES_REQUESTED** [review]
Thanks for your pull request! However, I suggest another solution.

File: tests/test_connection.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 class TestMessageBus(MessageBus):
     """Message bus for testing."""
+    __test__ = False
```
[the remark this answers is not in the document]
Comment (z3ntu) [reply tests/test_connection.py]: done, I used `MockMessageBus` which fits better imo

**Review by jkonecny12 — APPROVED** [review]
Looks good to me too!

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**